source/query-with-natural-language/query-with-natural-language.txt

```diff
@@ -0,0 +1,72 @@
+.. _query-natural-language:
+
+===========================
+Query with Natural Language
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+
+You can use MongoDB Compass to ask natural language questions about your
+data. Compass uses AI to generate filter queries and aggregations based 
+on the prompts you provide.
+
+Use Cases
+---------
+
+You may want to use natural language to query in Compass to:
+
+- Ask plain text questions about your data.
+- Create an initial query or aggregation pipeline that you can 
+  modify to suit your requirements.
+- Learn how to write complex queries with multiple aggregation stages.
+
+Behavior
+--------
+
+- Compass natural language querying is available starting in version 
+  ``1.40.x``.
+
+- Natural language querying utilizes `Azure Open AI 
+  <https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+  as its current provider. This provider may be subject to change in 
+  the future.
+
+- The MongoDB Compass natural language querying feature is on a rolling 
+  release schedule, with additional functionality and improvements planned 
+  for release by the end of October. As a result, some users may 
```

Comment:
Have we decided to not mention a specific time frame?

Reply:
Yes, we decided to remove any date reference (I pushed this write after I sent you the PR, sorry about that!).